platform/android/app/build.gradle

```diff
@@ -107,6 +107,7 @@ repositories {
 
 dependencies {
     implementation project(":polyOut")
+    implementation project(":PolyPodCoreAndroid:core")
```

Comment:
Same thought - for the _polyOut_ module, it appears to work without having an extra module it seems, would be nice if we had the same setup here.

Reply:
`core` module was wrapped in PolyPodCoreAndroid so that we are able to build and test it separately from PolyPodAndroid, the same way we can do with PolyPodCoreSwift; so one does not have to build PolyPodAndroid in order to build and test `core`.

I think I can name the project just `android`, since the actual module that is integrated in PolyPodApp is `core`.

Reply:
Pointed directly to `core` module instead of going through `PolyPodCoreAndroid`.